deploy/central/tasks/task-ods-build-python.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  name: ods-build-python
+spec:
+  description: ODS Build Python applications
+  params:
+    - default: "false"
+      description: Whether quality gate needs to pass.
+      name: sonar-quality-gate
+      type: string
+    - default: "false"
+      description: Whether to skip the SonarQube analysis or not.
+      name: sonar-skip
+      type: string
+    - default: ""
+      description: HTTPS proxy holds the hostname or IP address of your proxy server.
+      name: https-proxy
+      type: string
+    - default: ""
+      description: No Proxy
+      name: no-proxy
+      type: string
```

Comment:
Do we really need or want these proxy params? So far this is implicit in the image. I think we should either keep it implicit, or make a decision to always have it explicitly. I think explicit is very hard to manage, and the value is really something defined by cluster admins. I am in favour of implicit in this case, and not providing any params to adjust.